app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
```

Comment:
- [ ] Please make sure that your login and your sign-up views look like the design of the border around the input fields and the texts around the login and signup.

|Your design  | wireframe |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/74744787/216175609-e1379315-4f24-4785-a567-35778fcee506.png) | ![image](https://user-images.githubusercontent.com/74744787/216175673-b063a48f-680b-4c7b-99a9-9e7fb9705f07.png) |